install_all.sh

```diff
@@ -148,12 +153,24 @@ if [[ "$NEED_INSTALL_DOCKER_COMPOSE" == "YES" ]]; then
 fi
 
 echo "----------------------------------------------------"
-curl -k -L -o ui-bakery-on-premise_v2.17.1.tar.gz https://storageaccountrguib99d2.blob.core.windows.net/ui-bakery-cloud-on-premise/ui-bakery-on-premise_v2.17.1.tar.gz
+if [[ "$CUR_VERSION_UI_BAKERY" == "latest" ]]; then
+  file_name="ui-bakery-on-premise-vlatest.tar.gz"
+else
+  file_name="ui-bakery-on-premise_v${CUR_VERSION_UI_BAKERY}.tar.gz"
+fi  
+curl -k -L -o ${file_name} https://storageaccountrguib99d2.blob.core.windows.net/ui-bakery-cloud-on-premise/${file_name}
 
-mkdir ui-bakery-on-premise
-tar -xvf ui-bakery-on-premise_v2.17.1.tar.gz -C ui-bakery-on-premise
-cd ui-bakery-on-premise
+[ -d ./ui-bakery-on-premise ] || mkdir ui-bakery-on-premise
 
+echo "----------------------------------------------------"
+echo "Unpacking the archive....."
+tar -xvf ${file_name} -C ui-bakery-on-premise > /dev/null
```

Comment:
```suggestion
tar -xf ${file_name} -C ui-bakery-on-premise > /dev/null
```
-v - verbose, если убрать, всё ок

Reply:
Fixed